samples/dashboards/demo/aws-management/demo.details

```diff
@@ -0,0 +1,12 @@
+---
+name: AWS Management
+authors:
+  - Sebastian Bochan
+js_wrap: b
+tags:
+  - Highcharts Dashboards demo
+categories:
+  - General:
+        priority: 12
```

Comment:
Maybe it should go slightly higher in the hierarchy?

Reply:
Any suggestion?

Reply:
Perhaps next to the Project Management demo as they are somewhat similar? 

Maybe also worth considering if the demos could be split into separate catagories? I see there's several with 'sync' in the title that perhaps could go into a "Synchonization" (/Synchronisation?) category.

Reply:
good point! added to an internal discussion!

Reply:
Just set the priority as 11, until end of subcategories discussion